tools/tasks/seed/check.tools.ts

```diff
@@ -0,0 +1,30 @@
+import * as util from 'gulp-util';
+import * as runSequence from 'run-sequence';
+
+import { join } from 'path';
+
+let fs = require('fs');
```

Comment:
fyi if you revert that last commit it's back to using the import. maybe there is something obvious I was doing wrong there?